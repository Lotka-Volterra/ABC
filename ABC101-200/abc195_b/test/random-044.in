604093659 609752744 751376448
