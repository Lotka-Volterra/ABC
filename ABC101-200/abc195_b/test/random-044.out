1233 1243
